x-pack/plugins/reporting/public/components/report_listing.tsx

```diff
@@ -0,0 +1,315 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+// TODO: Remove once typescript definitions are in EUI
+declare module '@elastic/eui' {
+  export const EuiBasicTable: React.SFC<any>;
+  export const EuiTextColor: React.SFC<any>;
+}
+
+import moment from 'moment';
+import React, { Component } from 'react';
+import chrome from 'ui/chrome';
+import { toastNotifications } from 'ui/notify';
+// @ts-ignore: implicit any for JS file
+import { Poller } from '../../../../common/poller';
```

Comment:
Can you make a `.d.ts` file for this interface, or switch to typescript (looks like it has no imports so I think should be relatively straightforward).  Then can get rid of `poller: any` below.